kafka-transport-binding.md

```diff
@@ -0,0 +1,281 @@
+# Kafka Transport Binding for CloudEvents
```

Comment:
for consistency with the other specs, can you add: `  - Version 0.4-wip` to the title?

Reply:
yep - done